code/modules/mob/inventory_procs.dm

```diff
@@ -23,6 +23,13 @@
 	if(I)
 		I.equip_to_best_slot(src)
 
+/mob/proc/can_use_hand(hand_name)
```

Comment:
this is a bad name, because hands may be otherwise disabled, via clone damage, etc. I recommend just sticking to has_right_hand and has_left_hand